main.c

```diff
@@ -409,9 +426,15 @@ static void load_image(char *arg, struct swaylock_state *state) {
 
 	// Bash doesn't replace the ~ with $HOME if the output name is supplied
 	wordexp_t p;
+	while (strstr(image->path, "  ")) {
```

Comment:
Hmm, sorry if I missed something, but why are we searching for double-spaces?

Reply:
wordexp will condense multiple spaces so stuff like `swaylock --image "path  with  multiple  spaces  in  a   row"` _(and of course, it looks like GitHub is condensing them too)_ need to have the spaces escaped.

Reply:
Ehh, right